utils/mil_tools/pythonProgramFinder/pythonFinder.py

```diff
@@ -0,0 +1,59 @@
+import os
+import re
+
+# Joseph Brooks
+# Checks NaviGator for all python files
+print("Python Finder")
+print("Type directory path you want to search from")
+directory = input(": ")
```

Comment:
This program should take arguments not stdin to pick the folder. It should also default to the current directory if not specified.  Use ```sys.argv```